components/core/src/streaming_archive/writer/File.hpp

```diff
@@ -147,9 +151,9 @@ namespace streaming_archive { namespace writer {
          */
         const std::string& get_orig_path () const { return m_orig_log_path; }
         const boost::uuids::uuid& get_orig_file_id () const { return m_orig_file_id; }
-        const std::string& get_orig_file_id_as_string () const { return m_orig_file_id_as_string; }
+        const std::string get_orig_file_id_as_string () const { return boost::uuids::to_string(m_orig_file_id); }
```

Comment:
For `get_orig_file_id_as_string` and `get_id_as_string`
* Can remove `const` from the return type
* If these methods exist, the member can be removed right?